.github/workflows/build.yml

```diff
@@ -0,0 +1,42 @@
+name: Attach Build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 5
+
+      - name: Setup Java 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      - name: Build project
+        run: |
+          ./gradlew -i build
```

Comment:
`build` doesn't do all the native builds. 
Probably worth using `publishToMavenLocal` in case of Pull Requests only?

Reply:
Do you think it makes sense to pull out `native-build` out of `mavenPublish.gradle` and move it to either into `build.gradle` or its own file?

It also makes sense to create its own task.

Reply:
It will be a complex process, probably better for a follow-up PR. So far calling `publishToMavenLocal` will call the native build for iOS and Android, and if it succeeds, that will be enough.